src/adapters/api-blueprint-adapter.coffee

```diff
@@ -298,9 +297,9 @@ legacyResourcesFrom1AResource = (legacyUrlConverterFn, resource, sourcemap, opti
       legacyResource.model = {}
 
     legacyResource.resourceParameters = resourceParameters or []
-    legacyResource.actionParameters   = actionParameters or []
+    legacyResource.actionParameters   = getParametersOf(action, options) or []
 
-    legacyResource.parameters = actionParameters or resourceParameters or []
+    legacyResource.parameters = legacyResource.resourceParameters.concat(legacyResource.actionParameters)
```

Comment:
@pksunkara thing is more complicated after you look into metamorphoses as one of Apiary-libraries used only there to produce App-AST.

And how Apiary works with the AST is written in Apiary, and whenever you are about to change tests fixtures, you're possibly breaking Apiary.

Unfortunately not all things (resource params, action params, action url) are tested in metamorphoses at all.